lib/determine-basal/determine-basal.js

```diff
@@ -465,7 +477,7 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
         console.error("Limiting carb impact from",ci,"to",maxCI,"mg/dL/5m (",maxCarbAbsorptionRate,"g/h )");
         ci = maxCI;
     }
-    var remainingCATimeMin = 3; // h; duration of expected not-yet-observed carb absorption
+    var remainingCATimeMin = 3; // h; minimum duration of expected not-yet-observed carb absorption
```

Comment:
should this comment be separated above the line to match the other comments?

Reply:
No, this is an in-line more-detailed comment, of a style I use occasionally after short variable assignment lines etc.